src/flowsynth.py

```diff
@@ -613,11 +613,19 @@ def render(self, eventid):
             else:
                 lyr_ip = IPv6(src = src_host, dst = dst_host)
             lyr_eth = Ether(src = src_mac, dst = dst_mac)
+            # the 'payload' variable is type str [sic] so here we put the payload into a bytearray
+            # and pass it to scapy using Raw(). If we don't do this, we can end up with situations
+            # where bytes (e.g. entered with hex encoding) can get interpreted as multi-byte UTF8
+            # when passed to scapy which is not the desired behavior.
+            payload_bytes = bytearray()
+            payload_bytes.extend(map(ord,payload))
```

Comment:
If you want, these two lines can be condensed to one, eliminating the `extend`.

```python
payload_bytes = bytearray(map(ord, payload))
```

Relatedly, if any unicode characters ever find their way into the `payload` string, the above will raise an exception:
```python-traceback
>>> bytearray(map(ord, u'here→there'))
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ValueError: byte must be in range(0, 256)
```

Reply:
Hmm, good point ... by doing the map we lose unicode support (in Python3 at least).  We should handle this better instead of like this....